var/spack/repos/builtin/packages/r-hh/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RHh(RPackage):
+    """Support software for Statistical Analysis and Data Display (Second
+    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```

Comment:
```suggestion
    """Support software for Statistical Analysis and Data Display.
    
    Support software for Statistical Analysis and Data Display (Second
    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```

Reply:
I did not realize that was parsed that way. I see that first line gets uses by `module whatis`. Which spack commands use the short description?

I will use the one liner from the project page.

Thanks.

Reply:
I believe the module files Spack generates use the short description, while `spack info` uses the full description.